### ISSUE TITLE
`"resourcePageBlocks" was not found` error in custom theme.

### ISSUE THREAD

**Issue by knadh** [omeka/omeka-s#1930]
Hi. I'm trying to create a custom theme. I've cloned the default theme directory and have copied and customized `/view/omeka/site/item/browse.phtml` successfully.

However, copying [view/omeka/site/item/show.phtml](https://github.com/omeka/omeka-s/blob/develop/application/view/omeka/site/item/show.phtml) to the theme directory throws the following error. Is there anything that needs to be done for the theme to access `resourcePageBlocks`?

```
 Laminas\ServiceManager\Exception\ServiceNotFoundException
A plugin by the name "resourcePageBlocks" was not found in the plugin manager Laminas\View\HelperPluginManager

Details:

Laminas\ServiceManager\Exception\ServiceNotFoundException: A plugin by the name "resourcePageBlocks" was not found in the plugin manager Laminas\View\HelperPluginManager in /home/omeka/www/vendor/laminas/laminas-servicemanager/src/AbstractPluginManager.php:143
Stack trace:
#0 /home/omeka/www/vendor/laminas/laminas-view/src/Renderer/PhpRenderer.php(375): Laminas\ServiceManager\AbstractPluginManager->get()
#1 /home/omeka/www/vendor/laminas/laminas-view/src/Renderer/PhpRenderer.php(393): Laminas\View\Renderer\PhpRenderer->plugin()
#2 /home/omeka/www/themes/default/view/omeka/site/item/show.phtml(11): Laminas\View\Renderer\PhpRenderer->__call()
#3 /home/omeka/www/vendor/laminas/laminas-view/src/Renderer/PhpRenderer.php(505): include('/home/omeka/www...')
#4 /home/omeka/www/vendor/laminas/laminas-view/src/View.php(206): Laminas\View\Renderer\PhpRenderer->render()
...
```

**Comment by zerocrates** [comment]
If you're cloning from the default branch of a theme, you're seeing code that's designed for the develop branch of Omeka S, which shifts themes to a new "resource blocks" system that lets administrators configure and reorder what's shown on show pages for items and other resources. This system requires new code that's provided in the Omeka S develop branch (and will be released as part of Omeka S 4.0.0).

To write a theme that will work on currently-released versions of Omeka S, you should start from the latest tagged version of a theme instead.

**Comment by knadh** [comment]
Ah, thanks! Didn't realise that the tip had breaking changes in templates.
